:

```diff
@@ -0,0 +1,95 @@
+name: pandas-dev
+channels:
+  - https://conda.anaconda.org/conda-forge/
+dependencies:
+  # required
+  - numpy>=1.15
+  - python=3.6
+  - python-dateutil>=2.6.1
+  - pytz
+
+  # benchmarks
+  - asv
+
+  # building
+  - cython>=0.29.13
+
+  # code checks
+  - black=19.10b0
+  - cpplint
+  - flake8
+  - flake8-comprehensions>=3.1.0  # used by flake8, linting of unnecessary comprehensions
+  - flake8-rst>=0.6.0,<=0.7.0  # linting of code blocks in rst files
+  - isort  # check that imports are in the right order
+  - mypy=0.730
+  - pycodestyle  # used by flake8
```

Comment:
looks like you commited this file by accident, pls remove

Reply:
done, thanks